### DISCUSSION TITLE
Is there a way to set the outline of an ImageRun?

### DISCUSSION THREAD

**Discussion by thomaswright1993** [dolanmiu/docx#1560] [Q&A]
I have looked through the API and I'm fairly certain that there is no current way to add a border to an ImageRun. 
I have made a work around to draw a border on my images in browser using the canvas but that makes it permanent.
Is there a work around to setting the outline value for an image? (creating a manual xml file for it?)

These are the image settings I want to be able to configure.

<img width="304" alt="Word Image Outline Settings" src="https://user-images.githubusercontent.com/8586510/174703544-d4a4d910-e5d6-4bca-a2b6-f761b2dbbeb6.png">

Thanks :)

**Comment by thomaswright1993** [comment]
Have there been any changes with ImageRuns over the last year? I've revisited my limited solution quite a few times and I'm finding real issues with my code being useable in different circumstances.

**Reply by dolanmiu** [reply]
Not possible right now I am afraid. I have to do some investigation myself into it.
